Sources/third_party/Experimental/Complex.swift

```diff
@@ -0,0 +1,325 @@
+// A major part of the Complex number implementation comes from
```

Comment:
From open source policies:
> For files in the third_party dir, leave the existing headers in place. If you’ve modified a file, append “Google LLC” to the copyright line.

Reply:
for the "append Google LLC" part I added the following:
This is the original one in the original header:
```swift
//  Code in libc++ is dual-licensed under the MIT and UIUC/NCSA licenses.
//  Copyright © 2009-2017 contributors to the LLVM/libc++ project.
```
Added Google LLC:
```swift
//  Code in libc++ is dual-licensed under the MIT and UIUC/NCSA licenses.
//  Copyright © 2009-2017 contributors to the LLVM/libc++ project, Google LLC.
```

Reply:
Hmm, this is a bit different. We did not contribute to LLVM/libc++ project, so we probably shouldn't add Google LLC to it.

Reply:
I'm writing a new suggestion in a different comment.